<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPlaylistDialog</class>
 <widget class="QDialog" name="SettingsPlaylistDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="addSong_Button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QListWidget" name="song_listWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="emotion_Groupbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Choose Mood:</string>
   </property>
   <widget class="QRadioButton" name="excitement_radioButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Excitement</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="engagement_radioButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Engagement</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="boredom_radioButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>90</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Boredom</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="deleteSong_Button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>88</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_Button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
